packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -221,7 +225,6 @@ async function instantiateModule(
       },
     )
 
-    delete e.importee
```

Comment:
It seems like two modules that import the same dep will share the same error when importing error, if delete it here, one of the module can't get the `importee` info

Reply:
I'm not sure what you mean about `one of the module can't get the importee info`, aren't we the only one using the `importee` info.

But it's an interesting find that nodejs would shared error references like this. Maybe we should not pass the error to `logger.error` to fix this. And add a note about it.

Reply:
Actually, this is not a nodejs issue. If A and B import the same dep X, there will be a Map to store the X resolving promise in case X is resolved twice. Which means there will be only one resolving X error

https://github.com/vitejs/vite/blob/9697e641f9feb9e2d31f52b6add2648b88515425/packages/vite/src/node/ssr/ssrModuleLoader.ts#L41-L48

Reply:
Good idea. Done